.config/molecule/config.yml

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
Neat! I didn't know molecule supported a project-wide base config with overrides, I'll make sure to use this in the future :+1:

Reply:
Yeah it's a pretty powerful feature, it supports both overrides and merges of molecule configs.